encoding/protobuf/protobuf.go

```diff
@@ -5,7 +5,7 @@ import (
 	"io"
 	"io/ioutil"
 
-	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/proto" //nolint:staticcheck
```

Comment:
it might be a better option to use the golang.org package instead of ignoring the error ?

Reply:
we have an open ticket for this. Unfortunately it is not straightforward with just an upgrade because it is a breaking change.